node_modules/col-whiteboards/tests/util.js

```diff
@@ -306,26 +352,82 @@ var assertGetWhiteboardsFails = module.exports.assertGetWhiteboardsFails = funct
  * @param  {String}             title                           The updated title of the whiteboard
  * @param  {Number[]}           members                         The ids of the users that should be a member of the whiteboard
  * @param  {Function}           callback                        Standard callback function
- * @param  {Whieboard}          callback.whiteboard             The updated whiteboard
```

Comment:
A whiteboard sounds much better

![WiiiiBoard](http://cdn1-www.comingsoon.net/assets/uploads/2015/06/Back-to-the-Future-II-Hoverboard.jpg)